app/views/items/show.html.erb

```diff
@@ -0,0 +1,12 @@
+<p>Name: <%= @merchant_item.name %></p>
+<p>Description: <%= @merchant_item.description %></p
+
+<p>Current Selling Price: $<%= '%.2f' % (@merchant_item.unit_price/100.to_f) %></p>
+
+  <p><%= link_to "Update Item", "/merchants/#{@merchant.id}/items/#{@merchant_item.id}/edit", method: :get %></p>
+
+  <% flash.each do |type, msg| %>
```

Comment:
We could put this in views/layouts/application.html.erb so it works on all pages

Reply:
Good idea.  I'm not super familiar with putting things in the layouts, so I'll wait until our next meeting to implement that.  On that same thought train, might be a good idea to add some partials as well.